ui/apps/everest/src/pages/databases/lastBackup/LastBackup.tsx

```diff
@@ -13,12 +14,15 @@ export const LastBackup = ({ dbName, namespace }: LastBackupProps) => {
 
   const { data: pitrData } = useDbClusterPitr(dbName, namespace);
 
-  const lastBackup = backups[backups.length - 1];
-  const lastBackupDate = lastBackup?.created || new Date();
+  const finishedBackups = backups.filter(
+    (backup) => backup.completed && backup.state === BackupStatus.OK
+  );
+  const lastBackup = finishedBackups[finishedBackups.length - 1];
```

Comment:
this one can break if none was retrieved from filter. Let's do some sanity checking here

Reply:
@fabio-silva 
If none was retrieved from the filter the array will be empty and `lastBackup` will be `undefined`. I don't think it can break

Reply:
You're right, I mixed up filter with find, which retrieves undefined. Good to go